andrada12/assignment5/Graph.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package package2;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Set;
+import java.util.TreeSet;
+
+/**
+ *
+ * @author Vali
+ */
+public class Graph {
+   HashMap<Character, TreeSet<Character> > graph;
+    
+    public HashMap<Character, TreeSet<Character>> setGraph (ArrayList<String> dictionary ){

```

Comment:
so this method takes a dictionary (a list of strings) and turn it into a graph where each vertex is a character, right ?
it would be nice to have documentation for this method and the one below, even though the methods are clear